modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,10 +5,107 @@
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-665]
+== Release 6.6.5 (January 2022)
+
+Couchbase Server 6.6.5 was released in January 2022.
+This maintenance release includes bug fixes in multiple components.
+
+[#fixed-issues-665]
+=== Fixed Issues
```

Comment:
Can we mention that Log4J has been update to 2.17.1 please ?

Reply:
I've added.